.jenkins/sarus_run_test.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -e
+set -x
+
+# Set up the compute node environment
+module load daint-gpu
+module add /project/d107/install/modulefiles/
+module load cray-python gcloud sarus
+gcloud auth configure-docker
+
+
+# Set up the working directing for the c12 test
+cd examples
+. /project/d107/install/venv/sn_1.0/bin/activate
+python write_rundir.py ../tests/pytest/config/default.yml "./c12_test"
```

Comment:
This command is provided by `fv3config`.
```suggestion
write_run_directory ../tests/pytest/config/default.yml "./c12_test"
```

Reply:
Done.  Using write_run_directory from the updated fv3config module and python_write_rundir.py has been removed.